sympy/printing/lambdarepr.py

```diff
@@ -244,7 +244,7 @@ def blacklisted(self, expr):
 
     def doprint(self, expr):
         lstr = super(NumExprPrinter, self).doprint(expr)
-        return "evaluate('%s')" % lstr
+        return "evaluate('%s', truediv=True)" % lstr
```

Comment:
I'm not finding reference to `truediv` anywhere. Is `evaluate` the sympy function?